db/migrate/20210411033550_add_outfit_information.rb

```diff
@@ -0,0 +1,6 @@
+class AddOutfitInformation < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :mu, :string
+    add_column :users, :outfit, :string
+  end
+end
```

Comment:
What's the function of this? What's "outfit"?

Reply:
data that is useful to the organization. each student has an MU (like "1 BDE") and an outfit (like "F-2"). if someone is missing meetings, they use this information to find the people in charge of them in the corps.

Reply:
Ah, I see, this is the Corps information of an appropriate user.